### ISSUE TITLE
[Accessibility] Account selection dropdown - individual accounts only announced as "Switch account"

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/clients#12692]
### Steps To Reproduce

1. Use a screen reader
2. Open the account switching menu/dialog
3. Set focus to another account

### Expected Result

The account has visible text information (e.g. "hello@inclusivedesign24.org, bitwarden.com, locked") - this should be announced as part of the button's accessible name.

### Actual Result

The accounts are only announced with generic "Switch account: button". This is because the *actual* account info is currently mistakenly `aria-hidden="true"` https://github.com/bitwarden/clients/blob/196c1e1fa4189946d14b5aea468fb7d6aa94ed6d/apps/desktop/src/app/layout/account-switcher.component.html#L79

This makes the buttons useless for screen reader users, and violates WCAG 2.2 SC 2.5.3 Label in Name (at least)

### Screenshots or Videos

![Bitwarden desktop and NVDA screen reader speech viewer, showing that the focused button in the account dropdown which includes email address etc info visibly is only announced as 'Switch account: button'](https://github.com/user-attachments/assets/289293df-ca45-4fda-95ef-098df612d3ed)

### Operating System

Windows

### Operating System Version

11

### Installation method

Direct Download (from bitwarden.com)

### Build Version

2024.12.1

### Issue Tracking Info

- [x] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by SergeantConfused** [comment]
Hello @patrickhlauke,

Thank you for your report. I have flagged this to the Engineering department; please feel free to post additional information, such as screenshots or a screen video recordings, if you wish.

Thank you again,
